Makefile

```diff
@@ -34,7 +34,8 @@ CXX_FLAGS ?= -g
 SANITIZER ?= default
 SOURCEDIR = $(CURDIR)
 BUILDROOT = build
-BUILD     = $(BUILDROOT)/$(SANITIZER)
+SYSTEM    = $(shell uname -s)
+BUILD     = $(BUILDROOT)/$(SYSTEM)/$(SANITIZER)
```

Comment:
Looks like there might be a mixture of tabs and spaces in this file. Might be nice to be consistent about using one or the other if possible.

Reply:
Tabs at the start of the line are significant in `Makefile`s. There are some tabs at the start of line continuations which could be spaces. I don't think it is an issue but I can fix at some point if that is seen as a problem.